app/Header.test.tsx

```
@@ -27,63 +27,68 @@ describe("Header", () => {
+      const logo: JSX.Element = <p>{logoText}</p>;
+      const logoMock: MockedFunction<FC<LogoProps>> = mocked(Logo);
+      logoMock.mockReturnValue(logo);
 
-    render(
-      <Header tab={tab}>
-        <></>
-      </Header>
-    );
+      render(
+        <Header tab={tab}>
+          <></>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)